src/develop/develop.h

```diff
@@ -329,7 +329,10 @@ typedef struct dt_develop_t
   // ISO 12646-compliant colour assessment conditions
   struct
   {
-    GtkWidget *button; // yes, ugliness is the norm. what did you expect ?
+    GtkWidget *floating_window, *button; // yes, ugliness is the norm. what did you expect ?
```

Comment:
I really like the comment here :D

Reply:
Will do and replace with a todo comment referring to this refactoring issue: https://github.com/darktable-org/darktable/issues/18559